<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>qwm::QWidgetMap</class>
 <widget class="QWidget" name="qwm::QWidgetMap">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="mouseTracking">
   <bool>true</bool>
  </property>
  <property name="focusPolicy">
   <enum>Qt::StrongFocus</enum>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="qwm::util::QProgressIndicator" name="m_progress_indicator" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>25</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="m_widget_zoom" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>45</width>
     <height>170</height>
    </rect>
   </property>
   <widget class="QPushButton" name="m_button_zoom_in">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>25</width>
      <height>25</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>25</width>
      <height>25</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>25</width>
      <height>25</height>
     </size>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QSlider" name="m_slider_zoom">
    <property name="geometry">
     <rect>
      <x>12</x>
      <y>35</y>
      <width>21</width>
      <height>100</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>100</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>100</height>
     </size>
    </property>
    <property name="value">
     <number>99</number>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="m_button_zoom_out">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>135</y>
      <width>25</width>
      <height>25</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>25</width>
      <height>25</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>25</width>
      <height>25</height>
     </size>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>qwm::util::QProgressIndicator</class>
   <extends>QWidget</extends>
   <header>util/QProgressIndicator.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
